src/components/ProjectList.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <q-table
+    grid
+    title="Projects"
+    :rows="Object.values(projects)"
+    row-key="id"
+    hide-header
+    hide-bottom
+    virtual-scroll
+    :pagination="{ rowsPerPage: 0 }"
+  >
+    <template v-slot:item="props">
```

Comment:
Add a default render when we don't have any project, like:
`No projects, please create a new project to have one here ;)` (in grey italic)